bitnami/jenkins/README.md

```diff
@@ -302,6 +302,10 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 11.0.0
+
+This major release is no loger contais preinstalled plugins.
```

Comment:
```suggestion
This major release no longer contains preinstalled plugins.
```

I would also add if there are any expected breaking issues for those not using any plugin (if such a thing is possible). Also, we should give some upgrading steps or redirect to related Jenkins' documentation in case there may be any type of breaking changes.

Reply:
There aren't preinstalled plugins, if the user wants to have preinstalled plugin is in the [README.md of Jenkins](https://github.com/bitnami/containers/tree/main/bitnami/jenkins#installing-plugins). The plugins we installed was email and git, maybe some user missed them but only have to follow the documentation 🔼 .

Reply:
In that case, I would at least include a link to Jenkins' doc for those that were using plugins in a previous version.

Reply:
I just added a link to the official documentation to install plugins.